crew

```diff
@@ -64,6 +80,9 @@ rescue Docopt::Exit => e
     if ARGV[0] == '-V' or ARGV[0] == '--version' then
       puts CREW_VERSION
       exit 0
+    elsif ARGV[0] == '-L' or ARGV[0] == '--license' then
+      puts LICENSE
+      exit 0
     end
```

Comment:
I'm thinking this might be a bit more elegant:
```
case ARGV[0]
when '-V', '--version'
  puts CREW_VERSION
  exit 0
when '-L', '--license'
  puts LICENSE
  exit 0
end
```